vello/src/scene.rs

```diff
@@ -125,13 +125,37 @@ impl Scene {
         // For performance reason we cut off the filter at some extent where the response is close to zero.
         let kernel_size = 2.5 * std_dev;
 
-        let t = Transform::from_kurbo(&transform.pre_translate(rect.center().to_vec2()));
+        let shape: Rect = rect.inflate(kernel_size, kernel_size);
+        self.draw_blurred_rounded_rect_in(&shape, transform, rect, brush, radius, std_dev);
+    }
+
+    /// Draw a rounded rectangle blurred with a gaussian filter in `shape`.
+    ///
+    /// For performance reasons, `shape` should not extend more than approximately 2.5 times
+    /// `std_dev` away from the edges of `rect` (as any such points will not be perceptably painted to,
+    /// but calculations will still be performed for them).
+    ///
+    /// This method is a more advanced version of [`Self::draw_blurred_rounded_rect`].
```

Comment:
This comment is ok, but I think I might phrase it a little more directly:

``This method effectively draws the blurred rounded rectangle clipped to the given shape. If just the blurred rounded rectangle is desired without clipping, use the simpler [`Self::draw_blurred_rounded_rect`].``